clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -143,6 +145,88 @@ private void setupCoordinator(int retryBackoffMs, int rebalanceTimeoutMs, Option
                                                 mockTime);
     }
 
+    @Test
+    public void testMetrics() {
+        setupCoordinator();
+
+        metrics.sensor("heartbeat-latency").record(1.0d);
+        metrics.sensor("heartbeat-latency").record(2.0d);
+        metrics.sensor("heartbeat-latency").record(6.0d);
```

Comment:
Nit: To ensure that `max` metric is collected correctly, it might be good to change the order of those three (otherwise, max could be the same as "last value") ?